code/src/index.css

```diff
@@ -1,13 +1,152 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+/** @format */
+
+* {
+	box-sizing: border-box;
+	font-family: "Trispace", sans-serif;
+}
+
+.main-container {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	margin: 20px;
+}
+
+.form-container,
+.printed-message-container {
+	border: 1px solid black;
+	background-color: #f2f0f0;
+	flex-direction: column;
+	align-items: flex-start;
+	margin: 15px;
+	padding: 20px;
+	width: 300px;
+	box-shadow: 5px 5px;
+	word-wrap: break-word;
+}
+
+.form-title {
+	font-weight: 500;
+}
+
+.input-box {
+	width: 100%;
+	padding: 15px, 10px;
+	resize: none;
+	margin: 10px auto 10px auto;
+	height: 100px;
+	font-size: 14px;
+}
+
+.send-message-btn {
+	background-color: #f9adae;
+	border: none;
+	border-radius: 25px;
+	font-size: 14px;
+	padding: 10px 15px;
+	cursor: pointer;
+	margin-top: 10px;
+}
+
+.input-box {
+	outline-color: #f9adae;
+}
+
+.heart-emoji {
+	margin: 10px;
+}
+
+.printed-message-container {
+	background: white;
+	margin-top: 30px;
+}
+
+.printed-message {
+	font-size: 14px;
+}
+
+.heart-counting-date-wrapper {
+	display: flex;
+	flex-direction: row-reverse;
+	align-items: baseline;
+	justify-content: space-between;
+}
+
+.button-counting-wrapper {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+}
+
+.heart-button {
+	border: none;
+	border-radius: 50%;
+	padding: 16px;
+	color: gray;
+}
+
+.date-created {
+	color: gray;
+	align-self: flex-end;
```

Comment:
Smart to work with flex-end here. I decided to go with a position absolute and margin, and it worked to but this is a good approach to solve it in an even easier way. Great :)